dist/angular-promise-buttons.js

```diff
@@ -2,7 +2,7 @@ angular.module('angularPromiseButtons', [
 ]);
 
```

Comment:
Also: If you add "#13" in front of the commit messages, they will get linked automatically to the issue you created.